pkg/db/db.go

```diff
@@ -78,6 +79,11 @@ func NewClient(cacheDir string, quiet bool, opts ...Option) *Client {
 		opt(o)
 	}
 
+	// Add the schema version as a tag if the tag doesn't exist.
```

Comment:
Can you add a comment this is for backward compatibility? We want to remove it after a while.

Reply:
Added comment in https://github.com/aquasecurity/trivy/pull/6219/commits/91bcb0ab8ce79d6e58ec07600ee10fd2287fe4f7